### ISSUE TITLE
Add dart-define (or other flags) to dart builder?

### ISSUE THREAD

**Issue by gerardsimons** [kevmoo/peanut.dart#107]
I rely on some `--dart-define` parameters for the build process. Any way to pass these to dart build? Thanks!

**Comment by kevmoo** [comment]
Give me an example of something you pass in...so I can test

**Comment by gerardsimons** [comment]
Yes sir, here is the simplest example I could think of:

**main.dart**
```dart
import 'package:flutter/material.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
        title: 'Flutter Demo',
        theme: ThemeData(
          primarySwatch: Colors.blue,
        ),
        home: Scaffold(
          // body: IntegerStreamListWidget()
            body: Center(child: Text(const String.fromEnvironment("TEST_VAR", defaultValue: "<Missing Value>")))
        )
    );
  }
}
```

If you run it like this it should show a value 

`flutter run --dart-define TEST_VAR=123`

**Comment by tahseen0amin** [comment]
Any update on this ?

**Comment by tahseen0amin** [comment]
Yes. My Environment depends on it

**Comment by kevmoo** [comment]
Fixed in peanut v4.1.0
